src/pbkdf2_hmac_sha256_fmt_plug.c

```diff
@@ -43,7 +43,6 @@ john_register_one(&fmt_pbkdf2_hmac_sha256);
 
 #define FORMAT_LABEL            "PBKDF2-HMAC-SHA256"
 #define FORMAT_NAME		""
-#define BENCHMARK_LENGTH        0x107
```

Comment:
I guess this was a redefine of the same value coming from elsewhere?

```c
struct fmt_tests pbkdf2_hmac_sha256_common_tests[] = {
        /* Low iteration test vectors for comparison */
        {"$pbkdf2-sha256$1000$b1dWS2dab3dKQWhPSUg3cg$UY9j5wlyxtsJqhDKTqua8Q3fMp0ojc2pOnErzr8ntLE", "magnum"},
        {"$pbkdf2-sha256$1000$amkzNk9tOXJVZ043QTZ1dA$YIpbl0hjiV9UFMszHNDlJIa0bcObTIkPzT6XvhJPcMM", "bonum"},
```

Looks OK so far.

```
[solar@localhost run]$ GOMP_CPU_AFFINITY=0-255 GOMP_SPINCOUNT=10000 ../run/john -test -form=pbkdf2-hmac-sha256
Will run 256 OpenMP threads
Benchmarking: PBKDF2-HMAC-SHA256 [PBKDF2-SHA256 512/512 AVX512F 16x]... (256xOMP) DONE
Speed for cost 1 (iteration count) of 1000
Warning: "Many salts" test limited: 14/256
Many salts:     449756 c/s real, 1865 c/s virtual
Only one salt:  436906 c/s real, 1867 c/s virtual

[solar@localhost run]$ GOMP_CPU_AFFINITY=0-255 GOMP_SPINCOUNT=10000 ../run/john -test -form=pbkdf2-hmac-sha256
Will run 256 OpenMP threads
Benchmarking: PBKDF2-HMAC-SHA256 [PBKDF2-SHA256 512/512 AVX512F 16x]... (256xOMP) DONE
Speed for cost 1 (iteration count) of 1000
Warning: "Many salts" test limited: 14/256
Many salts:     441107 c/s real, 1865 c/s virtual
Only one salt:  436906 c/s real, 1870 c/s virtual

[solar@localhost run]$ GOMP_CPU_AFFINITY=0-255 GOMP_SPINCOUNT=10000 ../run/john -test -form=pbkdf2-hmac-sha256
Will run 256 OpenMP threads
Benchmarking: PBKDF2-HMAC-SHA256 [PBKDF2-SHA256 512/512 AVX512F 16x]... (256xOMP) DONE
Speed for cost 1 (iteration count) of 1000
Warning: "Many salts" test limited: 14/256
Many salts:     445390 c/s real, 1865 c/s virtual
Only one salt:  428740 c/s real, 1867 c/s virtual
```

Not sure why the difference, but perhaps it's just the sequential `set_key` overhead. This difference is much smaller than md5crypt's on this same machine, so I'm OK with 0x107.

Reply:
Yup, it's in `pbkdf2_hmac_common.h`. I got spurious reports of barely reaching 1 or 2% from a few of the formats but decided we'll just go with raw for all of them.